Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -1026,6 +1030,90 @@ public static void WhiteLabelLoginAndStartSession(string email, string password,
 
         #endregion
 
+        #region Account Linking
+
+        /// <summary>
+        /// Start an account linking process on behalf of the currently signed in player
+        /// When you want to link an additional provider to a player, you start by initiating an account link.The player can then navigate to the online link flow using the code_page_url and code, or the qr_code and continue the linking process.
+        /// For the duration of the linking process you can check the status using the CheckStatusOfAccountLinkingProcess method.
+        /// Returned from this method is the ID of the linking process, make sure to save that so that you can check the status of the process later.
+        /// https://ref.lootlocker.com/game-api/#start-account-link
+        /// </summary>
+        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerAccountLinkStartResponse</param>
+        public static void StartAccountLinkingProcess(Action<LootLockerAccountLinkStartResponse> onComplete)
+        {
+            if (!CheckInitialized())
+            {
+                onComplete?.Invoke(LootLockerResponseFactory.SDKNotInitializedError<LootLockerAccountLinkStartResponse>());
+                return;
+            }
+
+            var endpoint = LootLockerEndPoints.StartAccountLinkingProcess;
+
+            LootLockerServerRequest.CallAPI(endpoint.endPoint, endpoint.httpMethod, onComplete: (serverResponse) => { LootLockerResponse.Deserialize(onComplete, serverResponse); });
+        }
+
+        /// <summary>
+        /// Check the status of an ongoing account linking process
+        /// https://ref.lootlocker.com/game-api/#check-account-link-status
+        /// </summary>
+        /// <param name="LinkID">The ID of the account linking process which was returned when starting the linking process</param>
+        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerAccountLinkProcessStatusResponse</param>
+        public static void CheckStatusOfAccountLinkingProcess(string LinkID, Action<LootLockerAccountLinkProcessStatusResponse> onComplete)
```

Comment:
Same comment here as in the Unreal implementation with the naming

Reply:
Fixed